CONTRIBUTING.md

```diff
@@ -145,32 +261,490 @@ for review, which allows you to sidestep this issue.
 This is not a bulletproof method though, as OfBorg still does review requests even on draft PRs.
 ```
 
-## Backporting changes
+## How to backport pull requests
+[pr-backport]: #how-to-backport-pull-requests
+
+Once a pull request has been merged into `master`, a backport pull request to the corresponding `release-YY.MM` branch can be created either automatically or manually.
+
+### Automatically backporting changes
+
+> **Note**
+> You have to be a [Nixpkgs maintainer](./maintainers) to automatically create a backport pull request.
+
+Add the [`backport release-YY.MM` label](https://github.com/NixOS/nixpkgs/labels?q=backport) to the pull request on the `master` branch.
+This will cause [a GitHub Action](.github/workflows/backport.yml) to open a pull request to the `release-YY.MM` branch a few minutes later.
+This can be done on both open or already merged pull requests.
+
+### Manually backporting changes
+
+To manually create a backport pull request, follow [the standard pull request process][pr-create], with these notable differences:
+
+- Use `release-YY.MM` for the base branch, both for the local branch and the pull request.
+  > **Warning**
+  > Do not use the `nixos-YY.MM` branch, that is a branch pointing to the tested release channel commit
+
+- Instead of making manually making and committing the changes, use [`git cherry-pick -x`](https://git-scm.com/docs/git-cherry-pick) for each commit from the pull request you'd like to backport.
+  Either `git cherry-pick -x <commit>` when the reason for the backport is obvious (such as minor versions, fixes, etc.), otherwise use `git cherry-pick -xe <commit>` to add a reason for the backport to the commit message.
+  Here is [an example](https://github.com/nixos/nixpkgs/commit/5688c39af5a6c5f3d646343443683da880eaefb8) of this.
+
+  > **Warning**
+  > Ensure the commits exists on the master branch.
+  > In the case of squashed or rebased merges, the commit hash will change and the new commits can be found in the merge message at the bottom of the master pull request.
```

Comment:
Good to see this mentioned. It happened often PR's are opened against master/staging as well as with the backport before the change targetting master has been merged.